src/view/merkle-drop.njk

```diff
@@ -11,61 +11,218 @@
   {% if env.SHOW_MERKLE_DROP === 'true' %}
     <section class="section content-wrapper">
         <div id="root" />
+
     </section>
   {% endif %}
 
   <section class="section">
     <div class="container">
       <div class="content">
 
-      {% if env.SHOW_MERKLE_DROP === 'false' %}
-        <h1>Trustlines Merkle Drop</h1>
-      {% endif %}
+        {% if env.SHOW_MERKLE_DROP === 'false' %}
+          <h1>Trustlines Merkle Drop</h1>
+        {% endif %}
 
-        <p>The Merkle drop will:</p>
+        <div class="columns">
```

Comment:
The new text section below the claim app is slimmer. Is this intended or does it simply not matter? I just observed that you adjusted the column width for other pages as well. So you may want to shrink the app part as well? :thinking:

Reply:
Reacted to in the [below comment](https://github.com/trustlines-protocol/www.trustlines.foundation/pull/21#issuecomment-570174734) and can be resolved since it probably simply don't matter. Or more likely is not worth the work.